packages/@lwc/template-compiler/src/__tests__/fixtures/expression-complex/await/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "warnings": [
+        {
+            "code": 1083,
+            "message": "Unexpected compilation error: LWC1083: Error parsing template expression: expression must end with curly brace.",
```

Comment:
The error message seems incorrect in this case. It should be warning about the usage of `await`